app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -67,18 +71,18 @@
     private static final String NEWLINE = "\n";
     private static final Region region = Region.getRegion(Regions.US_EAST_1);
 
-    private String supportEmail;
+    private String supportEmail = "support@sagebridge.org";
```

Comment:
Spring won't mix Java-based configuration and XML-based configuration, so
moving that value out as a constant to the application-context.xml file
doesn't work. So this ended up in the code for the moment.

On Mon, Mar 2, 2015 at 1:55 PM, Dwayne Jeng <notifications@github.com>
wrote:

> In app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java
> <https://github.com/Sage-Bionetworks/BridgePF/pull/513#discussion_r25642400>
> :
>
> > @@ -67,18 +71,18 @@
> >      private static final String NEWLINE = "\n";
> >      private static final Region region = Region.getRegion(Regions.US_EAST_1);
> >
> > -    private String supportEmail;
> > +    private String supportEmail = "support@sagebridge.org";
>
> We should move this value to Spring, so we can still change it in config
> instead of code if we need it.
>
>
>  Also, the previous value was "Bridge (Sage Bionetworks) <
> support@sagebridge.org>". Was this change intentional?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Sage-Bionetworks/BridgePF/pull/513/files#r25642400>.
>